src/main/deltachat/webxdc.ts

```diff
@@ -101,7 +101,15 @@ export default class DCWebxdc extends SplitOut {
           accounts_sessions.push(accountId)
           ses.protocol.registerBufferProtocol(
             'webxdc',
-            async (request, callback) => {
+            async (request, __callback) => {
+              const respond = (response: Omit<ProtocolResponse, 'headers'>) => {
```

Comment:
The flexibility is not currently required. Headers are not currently set otherwise. And if I added this then you'd be able to override the already set headers this way, so I didn't want to bother and kept it simple.

Reply:
I think the `Omit<ProtocolResponse, 'headers'>` is confusing and I don't get why it should be needed here

Reply:
It states that you can't pass headers there because they'll be overridden.